docs/source/release_notes/release-notes.rst

```diff
@@ -50,8 +50,25 @@ develop
     *    - Type
          - Change
 
-    *    -
-         -
+    *    - |new| |improved| |metrics| |logs|
+         - Sweep metrics were reworked. Sweep now exposes metrics indicating the total number of cells examined, cells deleted, time spent sweeping, and time elapsed since sweep started on the current table that are updated after each iteration of sweep and separate metrics that are updated after each table is fully swept.
+           Additionally, sweep now exposes metrics tagged with table names that expose the total number of cells examined, cells deleted, time spent sweeping per iteration for each table separately.
+           Logs will also include the new timing information.
+           Sweep now exposes the following metrics with the common prefix ``com.palantir.atlasdb.sweep.metrics.SweepMetric.``:
```

Comment:
Which metrics (if any) are no longer reported?  Once this is merged and running somewhere you should upgrade sweep metrics dashboard to report either the old metric or the relevant new one.